wolftpm/tpm2.h

```diff
@@ -1667,7 +1667,7 @@ struct wolfTPM_winContext {
 };
 /* may be needed with msys */
 #ifndef TPM_E_COMMAND_BLOCKED
-#define TPM_E_COMMAND_BLOCKED (0x80284000)
+#define TPM_E_COMMAND_BLOCKED (0x80280400)
```

Comment:
If you search the code there are two other places in tpm2_wrap.c the "wrong" value is used. `0x80280400`

Reply:
I don't understand, when I grep for the old value nothing comes up
```
john@john-XPS-9320:~/Documents/wolfTPM$ grep -R "0x80284000"
john@john-XPS-9320:~/Documents/wolfTPM$ 
```
`0x80280400` is the right value according to this https://learn.microsoft.com/en-us/windows/win32/com/com-error-codes-6